css/style.css

```
@@ -0,0 +1,220 @@
+/* 
+Spacing system (px)
+2 / 4 / 8 / 12 / 16 / 24 / 32 / 48 / 64 / 80 / 96 / 128
+
+Font sizes (px)
+10 / 12 / 14 / 16 / 18 / 20 / 24 / 30 / 36 / 44 / 52 / 62 / 74 / 86 / 98
+*/
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+.wrapper {
+  width: 160rem;
+  margin: 0 auto;
+}
+
+/******************************/
+/* HEADER */
+/******************************/
+
+.header {
+  position: absolute;
+  display: flex;
+  width: 160rem;
+  height: 12rem;
+  justify-content: center;
+  align-items: center;
+  padding-left: 18rem;
+  padding-right: 18rem;
+  background: linear-gradient(to bottom, black -100%, transparent 90%);
+  gap: 0.8rem;
+}
+
+.logo-menu-bar {
+  display: flex;
+  justify-content: space-between;
+  gap: 22.8rem;
+  align-items: center;
+}
+
+.logo {
+  font-family: "Yeseva One", sans-serif;
+  text-decoration: none;
+  color: #fff;
+  text-transform: uppercase;
+  font-size: 3.6rem;
+  opacity: 0.9;
+}
+
+/******************************/
+/* NAVIGATION */
+/******************************/
+
+.main-nav-list {
+  list-style: none;
+  display: flex;
+  gap: 2.6rem;
+}
+
+.main-nav-link,
+.main-nav-link:link,
+.main-nav-link:visited,
+.main-nav-link:active {
+  padding: 4.8rem 1.2rem;
+  position: relative;
+  display: inline-block;
+  text-decoration: none;
+  color: #fff;
+  font-style: normal;
+  font-size: 2.2rem;
+  font-weight: 600;
+  transition: all 0.3s;
+  text-shadow: 1px 1px rgba(0, 0, 0, 0.3);
+  border-top: 0.38rem solid transparent;
+}
+
+.main-nav-link:hover,
+.main-nav-link:active {
+  opacity: 75%;
+  transition: 0.3s;
```

`transition` of what? You have to specify it
```suggestion
  transition: opacity 0.3s;
```

---

Fixed. Hope it's right.

---

The suggested change is right, so you can be sure 😄